apps/app/components/Preview/index.tsx

```diff
@@ -74,11 +74,11 @@ const PreviewContent = styled(Box)`
   img {
     display: inline;
   }
-  /*
+
   a {
     text-decoration: underline;
     color: #3182ce;
```

Comment:
It is recommended to change to theme color
```suggestion
    color: #4d51f3;
```